base/linalg/dense.jl

```diff
@@ -246,7 +246,7 @@ diag(A::AbstractMatrix, k::Integer=0) = A[diagind(A,k)]
 """
     diagm(v, k::Integer=0)
 
-Construct a matrix by placing `v` on the `k`th diagonal.
+Construct a matrix by placing `v` on the `k`th diagonal. This constructs a full matrix; if you want a storage-efficient version, use [`Diagonal`](@ref) instead.
```

Comment:
Should wrap lines at 92 characters. In this case wrapping after the semicolon would work well.